api/proto/teleport/legacy/types/types.proto

```diff
@@ -7801,3 +7803,11 @@ message AccessGraphAWSSync {
   // Integration is the integration name used to generate credentials to interact with AWS APIs.
   string Integration = 4 [(gogoproto.jsontag) = "integration,omitempty"];
 }
+
+// AccessGraphAzureSync is a configuration for Azure Access Graph service poll service.
+message AccessGraphAzureSync {
+  repeated string Regions = 1 [(gogoproto.jsontag) = "regions,omitempty"];
+  string SubscriptionID = 2 [(gogoproto.jsontag) = "regions,omitempty"];
+  string UmiClientId = 3 [(gogoproto.jsontag) = "regions,omitempty"];
```

Comment:
duplicate jsontags. 

What's the `UmiClientId`?

Reply:
Umi is User-managed Identity. It's the identity that's attached to the instance running the Discovery service in Azure. Could rename to UMIClientID, since I'm noticing we full-uppercase acronyms.